mkt/games/filters.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+
+import elasticsearch_dsl.filter as es_filter
+from elasticsearch_dsl import query, SF
+from rest_framework.filters import BaseFilterBackend
+
+import mkt
+from mkt.games.constants import GAME_CATEGORIES
+
+
+class DailyGamesFilter(BaseFilterBackend):
+    """
+    Randomly chooses 4 games, one from each featured game category, based off
+    of the current date such that the games are shuffled daily.
+    """
+    def filter_queryset(self, request, queryset, view):
+        # For each game category, create a query that matches the tag, orders
+        # randomly based on the day, and choose the first one.
+        game_category_query = (lambda cat:
+            query.Q(
+                'function_score',
+                # Consistently random based on the day.
+                functions=[SF('random', seed=datetime.datetime.now().day)],
```

Comment:
Are you sure you want `day`? Won't that mean the same 4 games on the first of each month? Something like `int(datetime.datetime.now().strftime('%Y%m%d'))` maybe?